src/stratigraphy/benchmark/score.py

```diff
@@ -203,34 +218,64 @@ def create_predictions_objects(
 
 def evaluate(
     predictions,
+    metadata_per_file: BoreholeMetadataList,
     ground_truth_path: Path,
     temp_directory: Path,
     input_directory: Path | None,
     draw_directory: Path | None,
 ):
     """Computes all the metrics, logs them, and creates corresponding MLFlow artifacts (when enabled)."""
-    predictions, number_of_truth_values = create_predictions_objects(predictions, ground_truth_path)
+    #############################
+    # Evaluate the borehole extraction metadata
+    #############################
+    metadata_metrics_list = evaluate_metadata_extraction(metadata_per_file, ground_truth_path)
+    metadata_metrics = metadata_metrics_list.get_cumulated_metrics()
+    document_level_metrics = metadata_metrics_list.get_document_level_metrics()
+
+    document_level_metrics.to_csv(
+        temp_directory / "document_level_metadata_metrics.csv"
+    )  # mlflow.log_artifact expects a file
+
+    # print the metrics
+    logger.info("Metadata Performance metrics:\n\n")
```

Comment:
why the double newline?

Reply:
I thought it was easier to read in the terminal. However, if you do not share this opinion, I am happy to roll back.

Reply:
Based on this I rolled back: https://github.com/swisstopo/swissgeol-boreholes-dataextraction/pull/80#discussion_r1764541075